cmd/zoekt-webserver/main.go

```diff
@@ -128,6 +144,7 @@ func main() {
 	index := flag.String("index", build.DefaultDir, "set index directory to use")
 	html := flag.Bool("html", true, "enable HTML interface")
 	enableRPC := flag.Bool("rpc", false, "enable go/net RPC")
+	enableSocketProxy := flag.Bool("socket", getEnvWithDefaultBool("SRC_ENABLE_SOCKET_PROXY", false), "proxy requests to /indexserver/ to <index dir>/indexserver.sock")
```

Comment:
```suggestion
	enableSocketProxy := flag.Bool("socket", getEnvWithDefaultBool("SRC_ENABLE_SOCKET_PROXY", false), "proxy requests to /indexserver/ via <index dir>/indexserver.sock")
```

Reply:
I also think the argument name `socket` isn't very descriptive. How about something like `indexserver_proxy`?

Reply:
Why do you also support the envvar? Doesn't seem consistent with how we do the other envvar flags.

Also agree socket is a confusing name. I'd call it `indexserver_proxy`?

Reply:
done done done.